tracer/src/Datadog.Trace/TraceContext.cs

```diff
@@ -49,6 +49,16 @@ public TraceContext(IDatadogTracer tracer)
 
         private TimeSpan Elapsed => StopwatchHelpers.GetElapsed(Stopwatch.GetTimestamp() - _timestamp);
 
+        /// <summary>
+        /// Gets or sets a collection of propagated internal Datadog tags,
+        /// formatted as "key1=value1,key2=value2".
+        /// </summary>
+        /// <remarks>
+        /// We're keeping this as the string representation to avoid having to parse.
+        /// For now, it's relatively easy to append new values when needed.
+        /// </remarks>
+        internal string DatadogTags { get; set; }
```

Comment:
Why not adding this to the interface and keep passing the interface (as the interface is internal I don't get why you're not adding it there)

Reply:
Good question, and I can't remember what I was thinking at the time 😄

Reply:
Looking again now, there were only 4 places that used the `ITraceContext` for no good reason, so I was just going to remove it entirely.

Reply:
I added the property to `ITraceContext` and moved those changes to a separate PR. See #2225.